.taskcluster.yml

```diff
@@ -0,0 +1,26 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $match:
+    '(tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"])':
+      taskId:
+        $eval: as_slugid("pr_task")
+      provisionerId: proj-getting-started
+      workerType: tutorial
+      payload:
+        maxRunTime: 3600
```

Comment:
We could probably adjust this down a bit, but it's hard to know how much until it initially works.